python/openvino_tokenizers/tokenizer_transformations.py

```diff
@@ -1,250 +1,184 @@
+import json
+from typing import Iterable, Tuple, List
+
+import numpy as np
 import openvino as ov
-from openvino import PartialShape, Type, op
+from openvino import PartialShape, Type
 from openvino import opset15 as opset
-from openvino.runtime.passes import AnyInput, Manager, Matcher, MatcherPass, WrapType
+from openvino.runtime.passes import AnyInput, Manager, Matcher, ModelPass, WrapType
 from openvino.utils.types import make_constant_node
 
 from . import _get_factory
+from .constants import PROCESSED_POST_PROCESSOR_NAME
 
+# TODO:
```

Comment:
This transformation is too cumbersome. I almost rewritten it. Please do not review this file, it's not the latest version. I will push the latest one shortly